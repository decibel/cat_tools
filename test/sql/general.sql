\set ECHO none

\i test/setup.sql

SELECT plan(
  0
  +1
);

SELECT hasnt_schema(
  '__cat_tools'
);

\i test/pgxntool/finish.sql

-- vi: expandtab ts=2 sw=2
