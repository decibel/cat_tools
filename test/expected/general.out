\set ECHO none
1..1
ok 1 - Schema __cat_tools should not exist
# TRANSACTION INTENTIONALLY LEFT OPEN!
